internal/store/builder.go

```diff
@@ -238,119 +238,264 @@ func availableResources() []string {
 }
 
 func (b *Builder) buildConfigMapStores() []*metricsstore.MetricsStore {
-	return b.buildStoresFunc(configMapMetricFamilies, &v1.ConfigMap{}, createConfigMapListWatch, b.useAPIServerCache)
+	return b.buildStoresFunc(
```

Comment:
nit: we should keep the same format as before

Reply:
My apologies. I had trouble fitting the line into the (24" 1080p) monitor - I use bigger fonts due to poor eyesight, Thought it may be an acceptable change.
I'll revert,